Packs/FeedFireEye/Integrations/FeedFireEye/FeedFireEye.py

```diff
@@ -0,0 +1,563 @@
+        }
+
+        return report
+
+
+class Client(BaseClient):
+    """Client to use in the FireEye Feed integration. Overrides BaseClient.
+
+    Args:
+        insecure (bool): False if feed HTTPS server certificate should be verified, True otherwise.
+        proxy (bool): False if feed HTTPS server certificate will not use proxies, True otherwise.
+        tags (list): The indicator tags.
+        tlp_color (str): Traffic Light Protocol color.
+    """
+
+    def __init__(self, public_key: str, private_key: str,
+                 polling_timeout: int = 20, insecure: bool = False, proxy: bool = False,
+                 tags: list = [], tlp_color: Optional[str] = None):
+        super().__init__(base_url=API_URL, verify=not insecure, proxy=proxy)
+        self.public_key = public_key
+        self.private_key = private_key
+        self._polling_timeout = polling_timeout
+        self.tags = tags
+        self.tlp_color = tlp_color
+
+    @staticmethod
+    def parse_access_token_expiration_time(expires_in: str) -> int:
+        """Computes the expiration time of the new fetched authentication time.
+
+        Args:
+            expires_in (str): Amount of time the authentication token will be valid according to the API.
+
+        Returns:
+            int. Epoch time that represents the expiration timeof the token.
+        """
+        try:
+            current_time = datetime.now()
+            expiration_time = current_time + timedelta(seconds=int(expires_in))
+            epoch_expiration_time = int(expiration_time.strftime('%s'))
+        except ValueError:
+            demisto.info('INFO - could not parse expiration time for access token.')
+            epoch_expiration_time = 0
+
+        return epoch_expiration_time
+
+    def fetch_new_access_token(self) -> str:
+        """Fetches new authentication token from the API.
+
+        Returns:
+            str. Authentication token.
+        """
+        response = self._http_request(
+            method='POST',
+            url_suffix='token',
+            data={'grant_type': 'client_credentials'},
+            auth=HTTPBasicAuth(self.public_key, self.private_key),
+            timeout=self._polling_timeout
+        )
+
+        auth_token = response.get('access_token')
+        expires_in = response.get('expires_in')
+        epoch_expiration_time = self.parse_access_token_expiration_time(expires_in)
+
+        demisto.setIntegrationContext(
+            {
+                'auth_token': auth_token,
+                'expiration_time': epoch_expiration_time
+            }
+        )
+
+        return auth_token
+
+    def get_access_token(self) -> str:
+        """Returns the current valid authentication token for the feed.
+
+        Returns:
+            str. Authentication token.
+        """
+        last_token_fetched_expiration_time = demisto.getIntegrationContext().get('expiration_time')
+        current_time = int(datetime.now().strftime('%s'))
```

Comment:
```suggestion
        current_time = int(datetime.now().timestamp())
```
isn't this what you are trying to do here?